src/components/Analyzer.tsx

```diff
@@ -713,7 +713,7 @@ export class AnalyzerView extends React.Component<AnalyzerViewProps, {
 
     // Draw frameCanvas to displayCanvas
     (this.displayContext as any).imageSmoothingEnabled = false;
-    this.displayContext.mozImageSmoothingEnabled = false;
+    (this.displayContext as any).mozImageSmoothingEnabled = false;
```

Comment:
mozImageSmoothingEnabled is actually deprecated, if you want instead you can just remove this line.

Reply:
Yes I can confirm this when I was testing analyzer deployment it was giving some problem,
https://github.com/xiph/awcy/issues/289#issuecomment-827102868